pkgs/tools/text/runiq/default.nix

```diff
@@ -15,6 +15,9 @@ rustPlatform.buildRustPackage rec {
     darwin.apple_sdk.frameworks.Security
   ];
 
+  cargoBuildFlags = [ "-Z avoid-dev-deps" ];
```

Comment:
I don't know if this is good enough, but please add a comment here to indicate the intention.

Reply:
Turns out, `avoid-dev-deps` is not needed actually, disabling the tests works.

Reply:
(it would avoid downloading the additional dependencies, but we probably don't care)